File: spring-boot-schedule/spring-boot-schedule-v2/src/main/java/org/luvx/schedule/v2/Application.java
package org.luvx.schedule.v2;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author Ren, Xie
 */
@MapperScan("org.luvx.schedule.v2.mapper")
@SpringBootApplication
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: spring-boot-case/fund/src/main/kotlin/org/luvx/Application.kt
package org.luvx

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

private val log = mu.KotlinLogging.logger {}

@MapperScan("org.luvx.fund")
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
    log.info { "启动成功......" }
}
